mantle/kola/harness.go

```diff
@@ -335,7 +340,23 @@ func parseDenyListYaml() error {
 			continue
 		}
 
-		fmt.Printf("⚠️  Skipping kola test pattern \"%s\":\n", obj.Pattern)
+		isSnoozed := false
+		if obj.SnoozeDate != "" {
+			snoozeDate, err := time.Parse(snoozeFormat, obj.SnoozeDate)
+			if err != nil {
+				return err
+			} else if today.After(snoozeDate) {
+				continue
+			}
+
+			fmt.Printf("🕒 Snoozing kola test pattern \"%s\" until %s\n", obj.Pattern, snoozeDate.Format("Jan 02 2006"))
```

Comment:
Should this be

```suggestion
			fmt.Printf("🕒 Snoozing kola test pattern \"%s\" until %s:\n", obj.Pattern, snoozeDate.Format("Jan 02 2006"))
```

to be consistent with the not-snoozed case?